Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingPassData.h

```diff
@@ -47,9 +47,11 @@ namespace AZ
                         ->Field("Thread Count X", &RayTracingPassData::m_threadCountX)
                         ->Field("Thread Count Y", &RayTracingPassData::m_threadCountY)
                         ->Field("Thread Count Z", &RayTracingPassData::m_threadCountZ)
-                        ->Field("Make Fullscreen Pass", &RayTracingPassData::m_makeFullscreenPass)
-                        ->Field("Max Ray Length", &RayTracingPassData::m_maxRayLength)
-                        ;
+                        ->Field("Make Fullscreen Pass", &RayTracingPassData::m_fullscreenDispatch)
+                        ->Field("FullscreenSizeSourceSlot", &RayTracingPassData::m_fullscreenSizeSourceSlot)
+                        ->Field("IndirectDispatch", &RayTracingPassData::m_indirectDispatch)
+                        ->Field("IndirectDispatchBufferSlot", &RayTracingPassData::m_indirectDispatchBufferSlot)
+                        ->Field("Max Ray Length", &RayTracingPassData::m_maxRayLength);
```

Comment:
Suggest to remove spaces.

Reply:
I've prepared https://github.com/o3de/o3de/pull/18265, but i've put it in a separate PR since is is an impactful change, and i'd like feedback on that regardless of the functionality in this PR.